### PR TITLE
Wizard recipe: libsixel-v1.8.6

### PR DESCRIPTION
**Pull request by johnnychen94**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: libsixel
* Version: v1.8.6

@staticfloat please review and merge.

### CODE REVIEW

**Comment by johnnychen94** [comment]
This is for https://github.com/JuliaImages/ImageInTerminal.jl/issues/45, I'm not sure if the name (libsixel) and version (v1.8.1) are chosen appropriately. I use `1.8.1` because it's build against sixel v1.8.1 release.

**Comment by giordano** [comment]
Would you be OK to require Julia v1.6 for the generated JLL package? That'd allow us to build for some extra platforms, including the new M1

**Comment by johnnychen94** [comment]
I'll be okay to this! Is there extra anything I need to do here?

File: L/libsixel/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/saitoha/libsixel.git", "5db717dfef6fa327cd4025e7352550f63d20699c")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libsixel/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --includedir=$WORKSPACE/destdir/include --libdir=$WORKSPACE/destdir/lib --enable-python=no
```
Comment (giordano) [inline L/libsixel/build_tarballs.jl:16]:
```suggestion
./configure --prefix=${prefix} \
    --build=${MACHTYPE} \
    --host=${target} \
    --includedir=${includedir} \
    --libdir=${libdir} \
    --enable-python=no
```

File: L/libsixel/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
```
Comment (giordano) [inline L/libsixel/build_tarballs.jl:23]:
```suggestion
platforms = supported_platforms(; experimental=true)
```

File: L/libsixel/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+products = [
+    LibraryProduct("libsixel", :libsixel)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
```
Comment (giordano) [inline L/libsixel/build_tarballs.jl:35]:
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6")
```

File: L/libsixel/build_tarballs.jl [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(; experimental=true)
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libsixel", :libsixel)
```
Comment (giordano) [inline L/libsixel/build_tarballs.jl:32]: There are also two executables, don't you care about them?
```suggestion
    LibraryProduct("libsixel", :libsixel),
    ExecutableProduct("img2sixel", :img2sixel),
    ExecutableProduct("sixel2png", :sixel2png),
```

Comment (johnnychen94) [reply L/libsixel/build_tarballs.jl:32]: Those are for command-line tools for terminal users, I think we can safely drop them and mimic the functionality inside `ImageInTerminal` and with all other JuliaImages functions (e.g., `save`). For the purpose of `ImageInTerminal`, only the shared library is needed.

Comment (johnnychen94) [reply L/libsixel/build_tarballs.jl:32]: Just to confirm if it's expected, so these executables are still shipped by libsixel_jll?

```julia
julia> using libsixel_jll

julia> run(`ls $(libsixel_jll.artifact_dir)/bin`)
img2sixel	libsixel-config	sixel2png
```

And the `products` part only controls how codes in the `_jll` package are generated?

Comment (giordano) [reply L/libsixel/build_tarballs.jl:32]: Correct

**Pull request merged**